application/tests/deployment-test.yaml

```diff
@@ -0,0 +1,11 @@
+suite: test deployment
+templates:
+  - charts/application/templates/deployment.yaml
+tests:
+  - it: make sure OAuth image is not included in the default rendered chart
+    asserts:
+      - isKind:
+          of: Deployment
+      - notEqual:
```

Comment:
Should rather use something like `notContains`, read up on https://github.com/quintush/helm-unittest/blob/master/DOCUMENT.md

Also add newline at end of file

Reply:
updated